### PR TITLE
Event streams webhook samples

### PR DESCRIPTION
**Pull request by mvazquezrius**
Event streams webhook samples for Java, Node.js and Python.

### CODE REVIEW

**Comment by sebas1208** [comment]
Hi @mvazquezrius can you remove the `package.json`, `package-lock.json`, `requirements.txt` and `pom.xml`. The platform will try to load those as code samples. Besides that, the other files will show fine in the docs.

Screenshot 2024-06-06 at 4 56 35 PM

*Ignore the zoom toolbar* 😄

File: event-streams/webhooks/python/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+2. Copy the public endpoint. It should look similar to `https://a8a06a2932bc.ngrok.app`.
+
+3. In a second terminal, start the sample server
+
+```bash
+export TWILIO_AUTH_TOKEN=<YOUR AUTH TOKEN>
+python3 -m venv venv
+source venv/bin/activate
+pip3 install -r requirements.txt
+uvicorn main:app --reload --port 8080
```
Comment (bermudezmt) [inline event-streams/webhooks/python/README.md]: I get an error with this part:
```
Will watch for changes in these directories: ['/Users/mbermudez/repos/api-snippets/event-streams/webhooks-server']
INFO:     Uvicorn running on http://127.0.0.1:8080 (Press CTRL+C to quit)
INFO:     Started reloader process [5973] using WatchFiles
ERROR:    Error loading ASGI app. Could not import module "main".
```

File: event-streams/webhooks-server/package.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "twilio-event-streams-samples-webhooks-node",
+  "version": "1.0.0",
+  "description": "Sample for receiving Event Streams webhooks with Node.js",
+  "main": "main.js",
```
Comment (bermudezmt) [inline event-streams/webhooks-server/package.json:5]: Does this and line 7 need to be updated since you renamed the JS file?

**Review by bermudezmt — CHANGES_REQUESTED** [review]

File: event-streams/webhooks/java/JavaWebhooksClientApplication.java [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+import jakarta.servlet.http.HttpServletRequest;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.web.bind.annotation.*;
+
+import java.net.URISyntaxException;
+
+@SpringBootApplication
+@RestController
+public class JavaWebhooksClientApplication {
```
Comment (bermudezmt) [inline event-streams/webhooks/java/JavaWebhooksClientApplication.java]: Hey, this also needs to be changed now as it's failing after the file rename:

```
INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  01:41 min
[INFO] Finished at: 2024-06-21T22:00:45-05:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.11.0:compile (default-compile) on project java-webhooks-client: Compilation failure
[ERROR] /Users/mbermudez/repos/api-snippets/event-streams/webhooks-server/webhooks-server.10.x.java:[16,8] class JavaWebhooksClientApplication is public, should be declared in a file named JavaWebhooksClientApplication.java
[ERROR] 
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
```

**Review by bermudezmt — CHANGES_REQUESTED** [review]

**Comment by bermudezmt** [comment]
A note for Matthew Setter: This is the [commit](https://github.com/TwilioDevEd/api-snippets/pull/1054/commits/13329eb02eb7c5a52468e8cef677f7e500454a09) with the working code samples and relevant README and requirements files.

**Comment by bermudezmt** [comment]
Thank you again for collaborating with me on this, @sebas1208 and @mvazquezrius . We will be closing the PR on account of our decision to host the content within the Twilio Blog.

**Pull request closed without merging**